types/node/fs.d.ts

```diff
@@ -599,7 +604,9 @@ declare module "fs" {
      * Synchronous lstat(2) - Get file status. Does not dereference symbolic links.
      * @param path A path to a file. If a URL is provided, it must use the `file:` protocol.
      */
-    export function lstatSync(path: PathLike): Stats;
+    export function lstatSync(path: PathLike, options?: lstatOptions & {
+      throwIfNoEntry?: boolean
```

Comment:
This option seems to be not available for NodeJs 14 but the type definitions here are still for nodejs 14.